infra/app/build-repository/main.tf

```diff
@@ -0,0 +1,51 @@
+locals {
+  project_name = "<PROJECT_NAME>"
+  app_name     = "<APP_NAME>"
+  region       = "<REGION>"
+
+  # Set project tags that will be used to tag all resources.
+  tags = {
+    project             = local.project_name
```

Comment:
There is already a project name defined in `infra/modules/common/outputs.tf  output "default_tags" {}`. Two thoughts/options related to this and one general one. 1. Remove project from "default_tags" or 2. Remove the `project = ` tag reference from here and then in  ` infra/modules/common/outputs.tf` create a new output “project_name” That you can use for project_name = module.common.project_name.

Reply:
yeah, i'm noticing that the placeholder string `<PROJECT_NAME>` needs to be replaced in multiple root modules, which i agree seems unnecessary.

i wasn't sure what the `common` module is for. is that supposed to be a way to define tfvars that are common across multiple root modules? so are you suggesting that we replace the hardcoded values of `owner=platform`, `project=template-application-nextjs`, and `repository` in `modules/common` with placedholders like `project=<PROJECT_NAME>`, and then reference that in the various root modules?

should we consider renaming `modules/common` to something like `modules/project-settings` or `modules/project-config` or `modules/project-info`? `common` isn't too descriptive

Reply:
related idea, what do you think about moving `common`/`project-config` to the `infra` folder, like `infra/project-config` rather than `infra/modules/project-config`? if it's the first thing a user of the platform needs to modify (to replace placeholders), it seems worth making it more visible rather than burying it amongst other modules that are generally reusable modules that don't need modification.

Reply:
Yeah common module is "the spot" to put anything that is a constant across all root modules. If it is more intuitive to move it up to the infra/X level I don't see a problem with that.